src/community/blocks.tsx

```diff
@@ -0,0 +1,347 @@
+import * as React from 'react';
+import {sprinkles} from '../sprinkles.css';
+import Stack from '../stack';
+import * as styles from './blocks.css';
+import {Text2, Text3, Text5, Text8} from '../text';
+import {vars} from '../skins/skin-contract.css';
+import Inline from '../inline';
+import Box from '../box';
+import ProgressBar from '../progress-bar';
+import classNames from 'classnames';
+
+import type StackingGroup from '../stacking-group';
+import type Image from '../image';
+import type Tag from '../tag';
+import type {RendersNullableElement} from '../utils/renders-element';
+
+interface BlockContentProps {
+    title?: string;
+    description?: string | Array<string>;
+}
+
+const BlockContent: React.FC<BlockContentProps> = ({title, description}) => {
+    const normalizedDescription =
+        description && Array.isArray(description) ? (
+            description.map((paragraph, i) => (
+                <Text2 regular color={vars.colors.textSecondary} key={i}>
+                    {paragraph}
+                </Text2>
```

Comment:
The intended behavior is to allow passing an array of description to the ValueBlock, so that multiple descriptions can be passed, and the reason why it is set to Text2 is because the Text2 specification is regular in Figma -> handoff -> Slots - text and color

The reason it is not also placed as an as='p' is that this customization option is only for the title and pretitle

Reply:
you are not responding my question.

What's the difference of sending an array of string vs just an string in `description`?
`description="some description"` vs `description={['some', 'description']}` are these equivanet? If they are, then why do we support both?

Reply:
we did it this way to facilitate passing the prop in case the programmer wanted to pass only one element

Reply:
Please, don't do that. We shouldn't support more than one way of doing the same thing

Reply:
@BrunoGuimaraes103, this is still pending, please remove this description Array

Reply:
unfortunately I can't remove this Array because it's part of the demand in these figma use cases:

<img width="539" alt="Captura de tela 2023-07-12 085419" src="https://github.com/Telefonica/mistica-web/assets/123959276/d6ad8636-cc58-4896-bee9-3d2d267485ed">
<img width="185" alt="Captura de tela 2023-07-12 085427" src="https://github.com/Telefonica/mistica-web/assets/123959276/76bba144-9c23-4fb2-80c0-bfa68b44b18d">